.travis.yml

```diff
@@ -37,11 +38,11 @@ env:
     # Enable end-to-end tests
     - E2E=no
   matrix:
-    - REQUIREMENTS=prod E2E=yes
+    - REQUIREMENTS=prod E2E=no
```

Comment:
For Travis to pass, we will have to add a version of invenio-search as a requirement.

Reply:
this will be fixed once https://github.com/inveniosoftware/invenio-rdm-records/pull/15 is merged

Reply:
Major: I don't think merging 15 is enough here. The python dependencies are installed in the `bootstrap.sh` script below and that script depends on there being a Pipfile. The Pipfile was removed in this PR. I thought it was because the Pipfile is generated by `invenio-cli`, but that's not the case yet. I think that's the missing piece.

This might be worth talking over

Reply:
Shelved: @ppanero will make this pass